packages/flutter_test/lib/src/binding.dart

```diff
@@ -127,6 +128,9 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
   @protected
   bool get checkIntrinsicSizes => false;
 
+  @override
+  bool disableAnimations = false;
```

Comment:
Apparently docs aren't inherited from mixins...

https://docs.flutter.io/flutter/flutter_test/TestWidgetsFlutterBinding/disableAnimations.html

Reply:
Tested locally, and it's not about mixins - it's about overriding a getter with a member variable.  Filed https://github.com/dart-lang/dartdoc/issues/1779